libandroid-navigation/src/main/AndroidManifest.xml

```diff
@@ -4,6 +4,11 @@
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
 
     <application>
+        <provider
+                android:name=".v5.provider.MapboxNavigationProvider"
+                android:authorities="${applicationId}.mapboxnavigationinitprovider"
+                android:exported="false"
+                android:initOrder="200"/>
```

Comment:
This is just priority. If the app has more than 1 content providers in what priority will these providers be executed. Telemetry provider has a priority of 100. so it will execute first. then nav